### ISSUE TITLE
ASIN tag not included in single or split M4B files, but included in single or split MP3 Files

### ISSUE THREAD

**Issue by ZLoth** [openaudible/openaudible#1462]
One thing that I noticed is that the Amazon ASIN number is not included in the M4B files (single chapter or whole book), but is included in MP3 files. Here is the relevant section from a probe from AudioBookShelf:

```
  "format": {
    "tags": {
      "asin": "1705290442",
    }
  }
```
[PROBE - M4B - Single Chapter.txt](https://github.com/user-attachments/files/17147762/PROBE.-.M4B.-.Single.Chapter.txt)
[PROBE - M4B - Whole Book.txt](https://github.com/user-attachments/files/17147763/PROBE.-.M4B.-.Whole.Book.txt)
[PROBE - MP3 - Single Chapter.txt](https://github.com/user-attachments/files/17147764/PROBE.-.MP3.-.Single.Chapter.txt)
[PROBE - MP3 - Whole Book.txt](https://github.com/user-attachments/files/17147765/PROBE.-.MP3.-.Whole.Book.txt)

**Comment by openaudible** [comment]
Should be fixed in latest beta and next release. Often listed as "CDEK": "B07D7JLN5B".
We use:

```ffprobe -hide_banner "Mission One.m4b" -show_streams -show_chapters -show_error -export_all 1 -show_format -print_format json```

**Comment by openaudible** [comment]
Should be fixed in 4.4.7: https://openaudible.org/

**Comment by ZLoth** [comment]
> Should be fixed in 4.4.7: https://openaudible.org/

Thank you very much!
